lib/Backend/Commands.hs

```diff
@@ -456,3 +468,11 @@ getEntryOrDir path =
                       \ entry or directory name."
                     , "Got: '" <> secret <> "'."
                     ]
+
+getBackend :: forall r. Member (Error CofferError) r => Config -> Maybe BackendName -> Sem r SomeBackend
+getBackend config backendNameMb = do
+  let backendName = maybe (mainBackend config) getBackendName backendNameMb
+  let backendsHashMap = backends config
+  case backendsHashMap !? backendName of
+    Just backend -> pure backend
+    Nothing -> throw BackendNotFound
```

Comment:
We should mention _which_ backend name was not found.

Reply:
Done